common/util.go

```diff
@@ -530,8 +530,12 @@ func CreateHistoryStartWorkflowRequest(
 	firstDecisionTaskBackoffSeconds := delayStartSeconds
 	if len(startRequest.GetCronSchedule()) > 0 {
 		delayedStartTime := now.Add(time.Second * time.Duration(delayStartSeconds))
-		firstDecisionTaskBackoffSeconds = backoff.GetBackoffForNextScheduleInSeconds(
+		var err error
+		firstDecisionTaskBackoffSeconds, err = backoff.GetBackoffForNextScheduleInSeconds(
```

Comment:
```suggestion
		firstDecisionTaskBackoffSeconds, err := backoff.GetBackoffForNextScheduleInSeconds(
```

Reply:
that declares a new `firstDecisionTaskBackoffSeconds` variable, rather than assigning it to the one outside the if block.

I really dislike Go's `:=` semantics :|  honestly I didn't notice this the first time, but thankfully Goland colors variables differently when they shadow others, so it was pretty obvious.